command/pr.go

```diff
@@ -560,19 +560,36 @@ func prMerge(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
+		var branchToSwitchTo string
 		if currentBranch == pr.HeadRefName {
+			branchToSwitchTo = repo.DefaultBranchRef.Name
 			err = git.CheckoutBranch(repo.DefaultBranchRef.Name)
 			if err != nil {
 				return err
 			}
 		}
 
-		err = git.DeleteLocalBranch(pr.HeadRefName)
+		localBranchExists := git.DoesLocalBranchExist(pr.HeadRefName)
+		if localBranchExists {
+			err = git.DeleteLocalBranch(pr.HeadRefName)
+			if err != nil {
+				err = fmt.Errorf("failed to delete local branch %s: %w", utils.Cyan(pr.HeadRefName), err)
+				return err
+			}
+		}
+
+		err = git.DeleteRemoteBranch(pr.HeadRefName)
 		if err != nil {
-			fmt.Fprintf(colorableErr(cmd), "%s Could not deleted local branch %s: %s\n", utils.Red("!"), utils.Cyan(pr.HeadRefName), err)
+			err = fmt.Errorf("failed to delete remote branch %s: %w", utils.Cyan(pr.HeadRefName), err)
 			return err
 		}
-		fmt.Fprintf(colorableOut(cmd), "%s Deleted local branch %s\n", utils.Red("✔"), utils.Cyan(pr.HeadRefName))
+
+		branchSwitchString := ""
+		if branchToSwitchTo != "" {
+			branchSwitchString = fmt.Sprintf(" and switched to branch %s", utils.Cyan(branchToSwitchTo))
+		}
+
+		fmt.Fprintf(colorableOut(cmd), "%s Deleted branch %s%s\n", utils.Red("✔"), utils.Cyan(pr.HeadRefName), branchSwitchString)
```

Comment:
This is kind of messy, but I haven't figured out a good way to do simple string interpolated stuff in go.